docs/en/engines/table-engines/mergetree-family/mergetree.md

```diff
@@ -1236,6 +1236,8 @@ Configuration markup:
 ```
 
 Connection parameters:
+* `endpoint` — AzureBlobStorage endpoint URL with container & prefix. Optionally can contain account_name if the authentication method used needs it. (http://azurite1:{port}/[account_name]{container_name}/{data_prefix}) or these parameters can be provided separately using storage_account_url, account_name & container. For specifying prefix, endpoint should be used.
```

Comment:
Could you please add an example how to set it for azure blob storage(not azurite)?

Reply:
addressed it